lingua_franca/lang/parse_de.py

```diff
@@ -81,6 +82,62 @@
 # The parameters are present in the function signature for API compatibility
 # reasons.
 
+def extract_duration_de(text):
+    """
+    Convert an german phrase into a number of seconds
+    Convert things like:
+        "10 Minuten"
+        "3 Tage 8 Stunden 10 Minuten und 49 Sekunden"
+    into an int, representing the total number of seconds.
+    The words used in the duration will be consumed, and
+    the remainder returned.
+    As an example, "set a timer for 5 minutes" would return
+    (300, "set a timer for").
+    Args:
+        text (str): string containing a duration
+    Returns:
+        (timedelta, str):
+                    A tuple containing the duration and the remaining text
+                    not consumed in the parsing. The first value will
+                    be None if no duration is found. The text returned
+                    will have whitespace stripped from the ends.
+    """
+
+    if not text:
+        return None
+
+    text=text.lower()
+    # die time_unit values werden für timedelta() mit dem jeweiligen Wert überschrieben
+    time_units = {
+        'microseconds' : 'mikrosekunden',
+        'milliseconds' : 'millisekunden',
+        'seconds'      : 'sekunden',
+        'minutes'      : 'minuten',
+        'hours'        : 'stunden',
+        'days'         : 'tage',
+        'weeks'        : 'wochen'
+    }
+
+    #### Einzahl und Mehrzahl
+    pattern = r"(?P<value>\d+(?:\.?\d+)?)(?:\s+|\-){unit}[ne]?"
+
+    #### Einstiegspunkt für Text-zu-Zahlen Konversion
+    #text = _convert_words_to_numbers_de(text)
```

Comment:
insert TODO here (many contributors' editors highlight TODOs, which will make this easier to spot when the time comes to integrate this function)

Reply:
Fixed but idk what to make of these

> This entry should've been there before, but you're adding support for de, so unless I need more coffee I don't think we need it anymore.